src/pe/mod.rs

```diff
@@ -57,6 +57,7 @@ pub struct PE<'a> {
     pub debug_data: Option<debug::DebugData<'a>>,
     /// Exception handling and stack unwind information, if any, contained in the PE header
     pub exception_data: Option<exception::ExceptionData<'a>>,
+    pub section_table: &'a [u8],
```

Comment:
can you add a doc comment here for the public field

Reply:
Done!